addons/music/fnc_getMusicPath.sqf

```diff
@@ -0,0 +1,28 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getMusicPath
+Description:
+    A function used to return the config path of a music file
+Parameters:
+    one string
+Example:
+    (begin example)
+    _configPath = ["LeadTrack01_F_Bootcamp"] call CBA_fnc_getMusicPath
+    (end example)
+Returns:
+    config
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+
+private ['_vars','_config','_duration'];
```

Comment:
again. Prefer not to use `private ARRAY`
also the `_vars` variable doesn't even exist.